File: #1010.cpp
// 다리 놓기
#include <iostream>
using namespace std;
int main()
{
    ios::sync_with_stdio(false);
    cin.tie();
    cout.tie();
    int T, N, M, cnt, temp;
    cin >> T;
    for(int i = 0; i < T; i++)
    {
        cnt = 1;
        temp = 1;
        cin >> N >> M;
        for(int j = M; j > M-N; j--)
        {
            cnt *= j;
            cnt /= temp++;
        }
        cout << cnt << "\n";
    }

    return 0;
}

File: #1010.py
# 다리 놓기
t = int(input())
for i in range(t):
    cnt = 1
    temp = 1
    n, m = map(int, input().split())
    for j in range(m, m - n, -1):
        cnt *= j
        cnt /= temp
        temp += 1
    print(int(cnt))